### PR TITLE
feature(bank-sdk): Skonto. Tests

### PR DESCRIPTION
**Pull request by ndubkov-distcotech**
feature(bank-sdk): Skonto. Tests

PP-770

### CODE REVIEW

File: bank-sdk/sdk/src/test/java/net/gini/android/bank/sdk/capture/skonto/usecase/GetSkontoEdgeCaseUseCaseTest.kt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        val useCase = GetSkontoEdgeCaseUseCase()
+        val paymentMethod = SkontoData.SkontoPaymentMethod.Cash
+        val result = useCase.execute(mockk(relaxed = true), paymentMethod)
+        assert(result == SkontoEdgeCase.PayByCashOnly)
+    }
+
+    @Test
+    fun `skonto last day edge case should be detected in case of due date tomorrow`() {
+        val useCase = GetSkontoEdgeCaseUseCase()
+        val dueDate = LocalDate.now().plusDays(0)
```
Comment (abolfazlimahdi) [inline bank-sdk/sdk/src/test/java/net/gini/android/bank/sdk/capture/skonto/usecase/GetSkontoEdgeCaseUseCaseTest.kt:41]: [Request] it seems like plusDays(0) is redundant (as 0 means add no days). Please verify if it is needed.

File: bank-sdk/sdk/src/test/java/net/gini/android/bank/sdk/capture/skonto/usecase/GetSkontoRemainingDaysUseCaseTest.kt [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package net.gini.android.bank.sdk.capture.skonto.usecase
+
+import org.junit.Test
+import java.time.LocalDate
+
+
+class GetSkontoRemainingDaysUseCaseTest {
+
+    @Test
+    fun `Skonto remaining days should be calculated correctly`() {
```
Comment (abolfazlimahdi) [inline bank-sdk/sdk/src/test/java/net/gini/android/bank/sdk/capture/skonto/usecase/GetSkontoRemainingDaysUseCaseTest.kt:10]: [Suggestion] What do you think about adding other tests for this method? you can verify today, and past as well.

**Review by abolfazlimahdi — APPROVED** [review]
Looks good. I just added two comments. Thank you 🥇

**Review by abolfazlimahdi — APPROVED** [review]
Looks good. Thank you!
